server/chat-commands/moderation.ts

```diff
@@ -422,7 +422,11 @@ export const commands: ChatCommands = {
 	part(target, room, user, connection) {
 		const targetRoom = target ? Rooms.search(target) : room;
 		if (!targetRoom) {
-			if (target.startsWith('view-')) return;
+			if (target.startsWith('view-')) {
+				connection.currentPages?.delete(target.slice(5));
+				if (!connection.currentPages?.size()) connection.currentPages = null;
```

Comment:
```suggestion
				if (!connection.currentPages?.size) connection.currentPages = null;
```

Ugh, I never remember if `set.size` is a function.